ckaznocha-marathon-resource.yml

```diff
@@ -0,0 +1,5 @@
+name: git
```

Comment:
I think this should be marathon 🤔?

Reply:
🤦‍♂ Looking at the example in the readme for some reason I thought `name` was for specifying the repo protocol. Fixed!

Reply:
That's actually a helpful feedback, maybe we need to update the Readme in a way that is easier to interpret.

Thanks.